scalene/scalene_profiler.py

```diff
@@ -1042,7 +1045,7 @@ def process_cpu_sample(
             Scalene.__stats.gpu_stats.gpu_mem_samples[fname][lineno].push(gpu_mem_used)
 
         # Now handle the rest of the threads.
-        for frame, tident, orig_frame in new_frames:
+        for frame, tident, orig_frame in new_frames + async_frames:
```

Comment:
You only use `async_frames` here, yet `total_frames` and the averages used to attribute time are computed across `new_frames`... is that right/consistent?